proto/services.proto

```diff
@@ -36,6 +36,11 @@ service PseudonymSystem {
 	rpc TransferCredential_EC (stream Message) returns (stream Message) {}
 }
 
+service CLCredentialInfo {
```

Comment:
I think this service unnecessary, maybe we can just add these two RPCs to the existing CL service.